config/dumps/sparql/construct_Sex.sparql

```diff
@@ -0,0 +1,11 @@
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+
+CONSTRUCT {
+	?s <http://n2o.neo/property/nodeLabel> "Sex" .
+}
+WHERE {
+VALUES ?s
+{obo:PATO_0000383 #female
+obo:PATO_000384 #male
```

Comment:
Are these two the only ones we need?

Reply:
Starting with those. I [asked Evan](https://github.com/kharchenkolab/cap-pipeline-config/issues/42#issuecomment-1096648552) if he would like to add others.